src/jdmtool/data_card/detect.py

```diff
@@ -49,7 +49,11 @@ def _open_usb_device(usbdev: USBDevice):
 
 
 SKYBOUND_VID_PID = (0x0E39, 0x1250)
-GARMIN_UNINITIALIZED_VID_PID = (0x091E, 0x0500)
+GARMIN_UNINITIALIZED_VID_PID = {
+    (0x091E, 0x0500), # "newer" 010-10579-20 
+    (0x091E, 0x0300), # "older" 011-01277-00
+}
+
 GARMIN_VID_PID = (0x091E, 0x1300)
 # If you reset the Garmin Programmer's EEPROM, you end up with a Cypress EZ-USB FX2
 FX2_VID_PID = (0x04B4, 0x8613)
```

Comment:
Might as well move this into `GARMIN_UNINITIALIZED_VID_PID`, to keep the logic simpler.